docs/runbooks/HCOMisconfiguredDescheduler.md

```diff
@@ -2,31 +2,31 @@
 
 ## Meaning
 
-A Descheduler is a Kubernetes application that causes the control plane to
+A descheduler is a Kubernetes application that causes the control plane to
```

Comment:
@jherrman  Descheduler is the name of the application so I think it should have a capital letter. WDYT?